src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { CourseCard } from './Components/CourseCard/CourseCard';
+import { SearchBar } from './Components/SearchBar/SearchBar';
+import { EmptyCourseList } from '../EmptyCourseList/EmptyCourseList';
+import { CourseData } from 'helpers/Types';
+import Button from 'common/Button/Button';
+import './Courses.css';
+
+export const Courses = ({ courseList, authorList }: any) => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const navigate = useNavigate();
+  const token = localStorage.getItem('token');
+  const handleSearch = (term: string) => {
+    setSearchTerm(term);
```

Comment:
You should be converting the search term to lowercase as well - right now you're only converting the course title / id

![image](https://github.com/elenapetra/react-app/assets/145835962/1bdb713f-ebe9-49f4-a833-85f321362162)

Reply:
fixed.